Scripts/script-StringLength.yml

```diff
@@ -0,0 +1,28 @@
+commonfields:
+  id: StringLength
+  version: -1
+name: StringLength
+script: |+
+  if (typeof args.str !== 'string') {
+      return {
+          Type: entryTypes.error,
+          Contents: 'Argument must be of type \'string\'. Not {0} type'.format(typeof args.str)
```

Comment:
`format` is python function if I am not mistaken. There is no such in JS. You don't have to check `args.str` to be string because all the args are strings when they passed from the CLI.